### ISSUE TITLE
一个nn.BatchNorm不能在一次前向中用多次。

### ISSUE THREAD

**Issue by BHM-Bob** [Oneflow-Inc/oneflow#10285]
## Summary
nn.BatchNorm不能在一次前向传播中用多次。
用oneflow复现一些对比学习架构的时候发现Check failed: mutable_consumer == nullptr 错误，查了下Issues感觉就是https://github.com/Oneflow-Inc/oneflow/issues/3516
但我现在用的是0.9.0+cu117，好像还没解决。。。

## Code to reproduce bug
把BatchNorm换成LayerNorm不会报错
```python
import oneflow as flow
import oneflow.nn as nn

x1 = flow.randn([32, 3]).to('cuda')
x2 = flow.randn([32, 3]).to('cuda')
label = flow.randn([32, 16]).to('cuda')

class GraphNet(nn.Graph):
    def __init__(self, loss_fn = nn.CrossEntropyLoss().to('cuda')):
        super().__init__()
        self.encoder_q = nn.Sequential(
            nn.Linear(3, 16),
            nn.BatchNorm1d(16)).to('cuda')
        self.encoder_q.train()
            
        self.loss_fn = loss_fn
        optimizer = flow.optim.SGD(self.encoder_q.parameters(), lr=1e-3)
        self.add_optimizer(optimizer, lr_sch=flow.optim.lr_scheduler.CosineAnnealingLR(optimizer, 8))
        
    def build(self, x1, x2, label):
        y1 = self.encoder_q(x1)
        y2 = self.encoder_q(x2)
        
        logits = y1*y2
        
        loss = self.loss_fn(logits, label)
        loss.backward()
        
        return logits, loss

g_net = GraphNet()
g_net(x1, x2, label)
```
现在用```cat([x1, x2], dim = 0)```来解决，但是这样batch size倍增。。。
